BUILDING.md

```diff
@@ -29,3 +29,18 @@ To build with only the TinkerPop tests:
 ```
 mvn clean install -Dtest.skip.tp=false -DskipTests=true
 ```
+
+## Building on Eclipse IDE
+Note that this has only been tested on Eclipse Neon.2 Release (4.6.2) with m2e (1.7.0.20160603-1933) and m2e-wtp (1.3.1.20160831-1005) plugin.
+
+To build without executing tests:
+1. Right-click on your project -> "Run As..." -> "Run Configurations..."
+2. On "Goals", populate with `install`
+3. Tick/check `Update Snapshots` and `Skip Tests`
+4. Before clicking "Run", make sure that Eclipse knows where `JAVA_HOME` is. On same window, go to "Environment" tab and click "New".
+5. Under "Name:", add `JAVA_HOME`
+6. Under "Value:", add the path where `java` is located
+7. Click "OK"
+8. Then click "Run"
+
+If you don't know how to find `java` on your environment, for *nix environments (Unix/Linux/macOS) you can do `which java`. For Windows you can do `for %i in (java.exe) do @echo. %~$PATH:i`.
```

Comment:
To find the Java binary in your environment, run the appropriate command for your operating system:

* Linux/macOS: `which java`
* Windows: `for %i in (java.exe) do @echo. %~$PATH:i`